modules/install/pages/ubuntu-debian-install.adoc

```diff
@@ -161,7 +161,7 @@ sudo apt-get update
 +
 [source,console,subs=+quotes]
 ----
-sudo dpkg -i ./[.var]_package-name_.deb
+sudo apt-get install -y ./[.var]_package-name_.deb
 ----
 +
 If `dpkg` reports any errors about missing dependencies, issue the following command to download and install those dependencies from the internet:
```

Comment:
This line and the following [source] block should be removed, since the instructions no longer say to invoke "dpkg". (apt-get will automatically download and install the required dependencies; that's the main reason I recommend using it instead of raw dpkg.)